packages/netlify-cms-backend-github/src/API.js

```diff
@@ -459,33 +470,82 @@ export default class API {
     return metadata;
   };
 
+  async migrateToVersion1(branch, metaData) {
+    // hard code key/branch generation logic to ignore future changes
+    const oldContentKey = branch.ref.substring(`refs/heads/cms/`.length);
+    const newContentKey = `${metaData.collection}/${oldContentKey}`;
+    const newBranchName = `cms/${newContentKey}`;
+
+    // create new branch and pull request in new format
+    const newBranch = await this.createBranch(newBranchName, metaData.pr.head);
+    const pr = await this.createPR(metaData.commitMessage, newBranchName);
+
+    // store new metadata
+    await this.storeMetadata(newContentKey, {
+      ...metaData,
+      pr: {
+        number: pr.number,
+        head: pr.head.sha,
+      },
+      branch: newBranchName,
+      version: CURRENT_METADATA_VERSION,
```

Comment:
This should be hard coded to '1' since we don't want it to change when someone updates `CURRENT_METADATA_VERSION` (see comment at the beginning of the function).

Reply:
ok, I will remove it.